### DISCUSSION TITLE
New Point-Blank Securly Fix

### DISCUSSION THREAD

**Discussion by Snfpy** [3kh0/ext-remover#634] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Not sure if it's just me but it stopped working, I think the extension ID changed, got a new link to the blocked page that works now though

chrome-extension://joflmkccibkooplaeoinecjbmdebglab/blocked.html?category=PGEgaHJlZj0iYWJvdXQ6YmxhbmsiIHRhcmdldD0iX2JsYW5rIiByZWw9Im9wZW5lciIgc3R5bGU9IiBiYWNrZ3JvdW5kOiB3aGl0ZTsgY29sb3I6IGJsYWNrOyBkaXNwbGF5OiBibG9jazsgaGVpZ2h0OiAzODBweDsgdHJhbnNmb3JtOiBzY2FsZSgyLjUpOyBkaXNwbGF5OiBmbGV4OyBqdXN0aWZ5LWNvbnRlbnQ6IGNlbnRlcjsgYWxpZ24taXRlbXM6IGNlbnRlcjsgIj48ZGl2PkNsaWNrIE1lPC9kaXY%2BIDwvYT4%3D

**Comment by amelia-bartlett** [comment]
if the extension id is different you can just substitute the old extension id for the new extension id
source: have dealt with the edge version of securly, which is literally the chrome version copy-pasted + a few manifest values changed and a new id

for reference:
chrome extension id is `iheobagjkfklnlikgihanlhcddjoihkg`
edge extension id is `dfkheabholbfmmehflddbknjimnjelda`
your extension id is `joflmkccibkooplaeoinecjbmdebglab`

if you want a working entrypoint for chrome or edge i have a website that points you to the correct one here:https://1984-was-a-warning-not-a-how-to-guide.bangsparks.com/ - note that it will not work for whatever version of the extension Snfpy has.

also chrome extension ids dont just change like that as that would break autoupdate (and they are derived from the extensions signing key iirc)
